tool/menu.lua

```diff
@@ -148,36 +148,57 @@ end
 
 local function randomizermenu()
   local selected = ui.select({
-    "Proficiency: " .. randomizer.proficiency,
-    "Unseeded",
-    "Seeded",
+    "New Seed on New Game: " .. randomizer.newgamenewseedui[randomizer.newgamenewseed],
```

Comment:
Suggestion of Ero was "New Seed when starting New Game", which might make this more understandable. Might be pretty long with the Auto option, but as that's only part of the menu, it shouldn't matter too much.